qiita_db/environment_manager.py

```diff
@@ -233,9 +233,10 @@ def make_environment(load_ontologies, download_reference, add_demo_user):
              'Demo', 'Qitta Dev', '1345 Colorado Avenue', '303-492-1984')""")
         conn.execute("""
             INSERT INTO qiita.analysis (email, name, description, dflt,
-                                        analysis_status_id)
+                                        analysis_status_id, portal_type_id)
```

Comment:
This is not correct. AFAIK, the qiita.analysis table doesn't have a porta_type_id column.